Hmf. missed one test that checked on this; updating.

Other failures were flaky tests;

Flaky test: https://github.com/moby/moby/issues/41338

```
=== RUN   TestDockerDaemonSuite/TestDaemonRestartUnlessStopped
    --- FAIL: TestDockerDaemonSuite/TestDaemonRestartUnlessStopped (6.59s)
        docker_cli_daemon_test.go:140: assertion failed: true (bool) != false (shouldRun bool): After daemon restart: container "exit" is running
```

Also flaky: https://github.com/moby/moby/issues/41276

```
=== RUN   TestDockerfileOutsideTheBuildContext
--- FAIL: TestDockerfileOutsideTheBuildContext (0.01s)
    internals_test.go:92: assertion failed: expected error to contain "Forbidden path outside the build context: ../../Dockerfile ()", got "forbidden path outside the build context: ../../Dockerfile ()"
```

And this one https://github.com/moby/moby/issues/23516

```
=== RUN   TestDockerSwarmSuite/TestAPISwarmNodeDrainPause
    --- FAIL: TestDockerSwarmSuite/TestAPISwarmNodeDrainPause (72.57s)
        docker_api_swarm_node_test.go:80: [d932b56ae14f5] joining swarm manager [d8e6e2b4f9f0a]@0.0.0.0:2477, swarm listen addr 0.0.0.0:2478
        docker_api_swarm_node_test.go:106: timeout hit after 1m0s: output: "001e834de0ef\nd172f836db13\n"
        poll.go:121: assertion failed: error is not nil: exit status 1
```

--- a/builder/dockerfile/internals_test.go
+++ b/builder/dockerfile/internals_test.go
@@ -50,7 +50,7 @@ func TestDockerfileOutsideTheBuildContext(t *testing.T) {
 	contextDir, cleanup := createTestTempDir(t, "", "builder-dockerfile-test")
 	defer cleanup()
 
-	expectedError := "Forbidden path outside the build context: ../../Dockerfile ()"
+	expectedError := "path outside the build context: ../../Dockerfile ()"
 	if runtime.GOOS == "windows" {
 		expectedError = "failed to resolve scoped path ../../Dockerfile ()"
 	}

--- a/builder/remotecontext/detect.go
+++ b/builder/remotecontext/detect.go
@@ -185,7 +185,7 @@ func FullPath(remote builder.Source, path string) (string, error) {
 		if runtime.GOOS == "windows" {
 			return "", fmt.Errorf("failed to resolve scoped path %s (%s): %s. Possible cause is a forbidden path outside the build context", path, fullPath, err)
 		}
-		return "", fmt.Errorf("Forbidden path outside the build context: %s (%s)", path, fullPath) // backwards compat with old error
+		return "", fmt.Errorf("forbidden path outside the build context: %s (%s)", path, fullPath) // backwards compat with old error
 	}
 	return fullPath, nil
 }

